MediaRecorder.html

```diff
@@ -33,15 +32,15 @@
 
     <p>This API attempts to make basic recording very simple, while still
     allowing for more complex use cases. In the simplest case, the application
-    instatiates a <code><a>MediaRecorder</a></code> object, calls <a href="#dom-
-    mediarecorder-start">start()</a> and then calls <a href="#dom-mediarecorder-
-    start">stop()</a> or waits for the MediaStreamTrack(s) to be ended. The
-    contents of the recording will be made available in the platform's default
-    encoding via the <a href="#dom-mediarecorder-ondataavailable">
-    ondataavailable</a> event. Functions are available to query the platform's
-    available set of encodings, and to select the desired ones if the author
-    wishes. The application can also choose how much data it wants to receive at
-    one time. By default a <a
+    instantiates a <code><a>MediaRecorder</a></code> object, calls
+    <a href="#dom-mediarecorder-start">start()</a> and then calls
+    <a href="#dom-mediarecorder-stop">stop()</a> or waits for the
+    MediaStreamTrack(s) to be ended. The contents of the recording will be made
+    available in the platform's default encoding via the <a
+    href="#dom-mediarecorder-ondataavailable">ondataavailable</a> event.
+    Functions are available to query the platform's available set of encodings,
+    and to select the desired ones if the author wishes. The application can
+    also choose how much data it wants to receive at one time. By default a <a
```

Comment:
I don't see any changes here. Not sure about this spec, but in other specs I know we sometimes try to leave formatting alone, or at least do it in separate PRs since it makes git blame harder to follow. YMMV